modules/ROOT/pages/installation.adoc

```diff
@@ -7,3 +7,472 @@ endif::[]
 Install API Community Manager with these instructions.
 
 Installed as a Salesforce package.
+
+API Community Manager 
+Setup Guide
+ (RC Version)
+
+Organization Installation Requirements - prereqs???
+Installing API Community Manager 
+Update API Community Manager - move this???
+Create a Community
+API Community Manager Components
+Publish API Community Manager
+Configure the Guest User Access
+Configure the Registered Users
+Configure Api Community Manager
+Set permissions for Unauthenticated users
+How to enable external users to register in your Community
+Define moderation rules for forums
+How to setup an Approval Process for new Community Users
+Customize Login and Registration Pages
```

Comment:
this should be in the branding and cusotmization section, so just remove it from here

Reply:
Removed.